packages/expo-modules-autolinking/scripts/ios/project_integrator.rb

```diff
@@ -105,7 +105,12 @@ def self.modules_providers_group(project, autocreate = false)
     def self.set_autolinking_configuration(project)
       project.native_targets.each do |native_target|
         native_target.build_configurations.each do |build_configuration|
-          configuration_flag = "-D #{CONFIGURATION_FLAG_PREFIX}#{build_configuration.name.upcase}"
+          if build_configuration.name.upcase.include? "RELEASE"
```

Comment:
I'm wondering if checking `DEBUG` wouldn't be better than `RELEASE` 🤔 Also, as an alternative I was considering using this method: https://github.com/CocoaPods/Xcodeproj/blob/b658adc5b74cecc892a55d53c04f02345cb4f64b/lib/xcodeproj/project/object/build_configuration.rb#L63
What do you think? May I ask you to try it?

Reply:
Makes sense, I'll have a look tomorrow!